docs/the_black_code_style/future_style.md

```diff
@@ -52,25 +52,37 @@ tracked in [this issue](https://github.com/psf/black/issues/2188).
 
 ### Removing trailing newlines after code block open
 
-_Black_ will remove trailing newlines after code block openings. That means that the
-following code:
+_Black_ will remove trailing newlines after code block openings. For example:
 
 ```python
 def my_func():
 
     print("The line above me will be deleted!")
-
-    print("But the line above me won't!")
 ```
 
-Will be changed to:
+will be changed to:
 
 ```python
 def my_func():
     print("The line above me will be deleted!")
-
-    print("But the line above me won't!")
 ```
 
 This new feature will be applied to **all code blocks**: `def`, `class`, `if`, `for`,
 `while`, `with`, `case` and `match`.
+
+### Improved parentheses management
```

Comment:
Also probably worth mentioning that long return annotations will now be moved to their own line surrounded by parens (black used to just them alone violating the line length limit ;-;)

---

```python
def foo() -> (int):
    ...
```
will be changed to:
```python
def foo() -> int:
    ...
```
```python
def foo() -> intsdfsafafafdfdsasdfsfsdfasdfafdsafdfdsfasdskdsdsfdsafdsafsdfdasfffsfdsfdsafafhdskfhdsfjdslkfdlfsdkjhsdfjkdshfkljds:
    return 2
```
will be changed to:
```python
def foo() -> (
    intsdfsafafafdfdsasdfsfsdfasdfafdsafdfdsfasdskdsdsfdsafdsafsdfdasfffsfdsfdsafafhdskfhdsfjdslkfdlfsdkjhsdfjkdshfkljds
):
    return 2
```

---

I'm a fan of examples, so I also suggested the return annotation one.

Reply:
Nice suggestion, will do